tests/integration/targets/dcnm_vrf/tests/dcnm/query.yaml

```diff
@@ -90,6 +90,23 @@
     - 'result.response[0].attach[1].vlan_id == 500'
     - 'result.response[0].vrf_name == "ansible-vrf-int1"'
 
+- name: QUERY - sleep for 10 seconds for DCNM to completely update the state
+  wait_for:
+    timeout: 10
+
+- name: QUERY - Query the VRF without a config element and verify the VRF status
+  cisco.dcnm.dcnm_vrf:
+    fabric: "{{ ansible_it_fabric }}"
+    state: query
+  register: result
+
+- assert:
+    that:
+    - 'result.changed == false'
+    - 'result.response[0].fabric == "{{ ansible_it_fabric }}"'
+    - 'result.response[0].vrfName == "ansible-vrf-int1"'
+    - 'result.response[0].vrfStatus == "DEPLOYED"'
```

Comment:
Does the `vrfStatus` get returned if you query WITH the `config:` element?  Same question for network.

Reply:
It should if it does not.

Reply:
No, the status for both vrf and network comes only when we enquire about "all" the vrf/network present if specified with a specific vrf/network name the status is not populated.

Reply:
Why is that?  Is there no way to get the status when a user only wants to know about a specific set of vrfs or networks?  Seems like this is something we should support.